.github/workflows/ci.yml

```diff
@@ -455,3 +455,99 @@ jobs:
           # is merged and the package becomes generally available.
           SELINUX: Permissive
         run: vagrant up --provision-with=selinux,install-runc,test-cri
+
+  integration-macos:
+    name: macOS Integration
+    runs-on: macos-10.15
+    timeout-minutes: 30
+    needs: [project, linters, protos, man]
+
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: '1.13.15'
+
+      - name: Set env
+        shell: bash
+        run: |
+          echo "::set-env name=GOPATH::${{ github.workspace }}"
+          echo "::add-path::${{ github.workspace }}/src/github.com/containerd/containerd/bin"
+
+      - name: Checkout containerd
+        uses: actions/checkout@v2
+        with:
+          path: src/github.com/containerd/containerd
+
+      - name: Install Build Deps
+        shell: bash
+        run: |
+          cd src/github.com/containerd/containerd
+          script/setup/install-dev-tools
+      - name: Install runtime
+        shell: bash
+        run: |
+          GO111MODULE=on go get -v github.com/ukontainer/runu
```

Comment:
Please pin a specific version

Reply:
addressed this temporary.  I will update with a proper version tag later.

Reply:
done.